app/lib/features/space/providers/space_navbar_provider.dart

```diff
@@ -3,6 +3,7 @@ import 'package:acter/common/providers/space_providers.dart';
 import 'package:acter/features/events/providers/event_providers.dart';
```

Comment:
this is the heart of it all:

Instead of having this await for all the internal states and thus fails if any of inner ones fails - in particular the space relations - this is now a list that is automatically updated with the proper tabs once they are loaded. This lowers the time-to-first-load and removes the need to have a full-space error message on failures.